.github/workflows/deploy.yml

```diff
@@ -182,7 +182,7 @@ jobs:
         uses: ./.github/workflows/composite/npm
         with:
           registry: 'https://npm.pkg.github.com'
-          scope: '@NeuraLegion'
+          scope: '@BrightSec'
```

Comment:
Irrelevant changes; moreover, there is no such scope. Please revert back.

Reply:
Dropped